### PR TITLE
Documenting the CI Results Explorer

### PR DESCRIPTION
**Pull request by MargotLepizzera**
### What does this PR do?

This PR documents the new CI Results Explorer.

### Motivation

Announcement of the feature.

### Preview

https://docs-staging.datadoghq.com/margot.lepizzera/ci-results-explorer/synthetics/cicd_testing/ci_results_explorer
https://docs-staging.datadoghq.com/margot.lepizzera/ci-results-explorer/synthetics/ci/?tab=apitest#in-datadog-application
https://docs-staging.datadoghq.com/margot.lepizzera/ci-results-explorer/synthetics/troubleshooting/#cicd-testing

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/synthetics/ci.md [hunk 1]
```diff
@@ -712,3 +712,4 @@ You can also see the results of your tests listed on your Datadog test details p
 [3]: https://github.com/TooTallNate/node-proxy-agent
 [4]: /api/v1/synthetics/#get-test
 [5]: /synthetics/testing_tunnel/
+[6]: /synthetics/cicd_testing
```
Comment (MargotLepizzera) [inline content/en/synthetics/ci.md:715]:
```suggestion
[6]: /synthetics/cicd_testing/ci_results_explorer
```

File: content/en/synthetics/troubleshooting/_index.md [hunk 2]
```diff
@@ -138,6 +138,12 @@ This might mean your private location is unable to reach the endpoint your API t
 
 {{< img src="synthetics/timeout.png" alt="API test on private location timing out" style="width:100%;" >}}
 
+## CI/CD Testing
+
+### I'm not seeing any CI metadata
```
Comment (MargotLepizzera) [inline content/en/synthetics/troubleshooting/_index.md:143]:
```suggestion
### I'm not seeing any CI metadata in my CI Results Explorer
```

File: content/en/synthetics/ci.md [hunk 3]
```diff
@@ -691,17 +691,17 @@ To learn how to get started using the testing tunnel, see the [Synthetics testin
...
 
 You can identify what caused a test to fail by looking at the execution logs and searching for causes of the failed assertion:
 
-{{< img src="synthetics/ci/failed_test_result.png" alt="Failed Test Result" style="width:80%;">}}
+{{< img src="synthetics/ci/failed_test_result.png" alt="Failed Test Result" style="width:100%;">}}
 
 ### In Datadog application
 
-You can also see the results of your tests listed on your Datadog test details page:
+You can also see the results of your tests listed on the [CI Results explorer][6] as well as on your individual test details page:
```
Comment (alai97) [inline content/en/synthetics/ci.md:702]:
```suggestion
You can also see the results of your tests listed on the [CI Results explorer][6] as well as on your individual test details page:
```
```suggestion
You can also see your test results listed in the [CI Results Explorer][6] and on test details pages:
```

File: content/en/synthetics/cicd_testing/ci_results_explorer.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+The panel on the left side of the page lists several facets you can use to search through your batches:
+
+* **Batch** facets allow you to filter on attributes of your batches:
+
+| Facet            | Description                                                 |
+|------------------|-------------------------------------------------------------|
+| `Summary Status` | The status of the batch: `Passed`, `Failed`, `In Progress`. |
+| `Duration`       | The overall duration of the batch.                          |
+| `ID`             | The id of the batch.                                        |
```
Comment (alai97) [inline content/en/synthetics/cicd_testing/ci_results_explorer.md:47]:
```suggestion
| `ID`             | The batch ID.                                        |
```

File: content/en/synthetics/cicd_testing/ci_results_explorer.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+| Facet          | Description                                 |
+|----------------|---------------------------------------------|
+| `CI Provider`  | The CI provider associated with the batch.  |
+| `Pipeline URL` | The pipeline URL associated with the batch. |
+
+**Test result** facets allow you to filter on attributes of the test results executed as part of your batches:
+
+| Facet            | Description                                                                                             |
+|------------------|---------------------------------------------------------------------------------------------------------|
+| `Execution Rule` | The execution rule associated with the test result of the batch: `Blocking`, `Non Blocking`, `Skipped`. |
```
Comment (alai97) [inline content/en/synthetics/cicd_testing/ci_results_explorer.md:67]:
```suggestion
| `Execution Rule` | The execution rule associated with the test result of the batch: `Blocking`, `Non Blocking`, or `Skipped`. |
```

File: content/en/synthetics/cicd_testing/ci_results_explorer.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+### Facets and tags
+
+The panel on the left side of the page lists several facets you can use to search through your batches:
+
+* **Batch** facets allow you to filter on attributes of your batches:
+
+| Facet            | Description                                                 |
+|------------------|-------------------------------------------------------------|
+| `Summary Status` | The status of the batch: `Passed`, `Failed`, `In Progress`. |
```
Comment (alai97) [inline content/en/synthetics/cicd_testing/ci_results_explorer.md:45]:
```suggestion
| `Summary Status` | The status of the batch: `Passed`, `Failed`, or `In Progress`. |
```

File: content/en/synthetics/cicd_testing/ci_results_explorer.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+| `Pipeline URL` | The pipeline URL associated with the batch. |
+
+**Test result** facets allow you to filter on attributes of the test results executed as part of your batches:
+
+| Facet            | Description                                                                                             |
+|------------------|---------------------------------------------------------------------------------------------------------|
+| `Execution Rule` | The execution rule associated with the test result of the batch: `Blocking`, `Non Blocking`, `Skipped`. |
+| `Fast Retries`   | The number of fast retries associated with the test result of the batch.                                |
+| `Location`       | The location associated with the test result of the batch.                                              |
+| `Test ID`        | The test id associated with the test result of the batch.                                               |
```
Comment (alai97) [inline content/en/synthetics/cicd_testing/ci_results_explorer.md:70]:
```suggestion
| `Test ID`        | The test ID associated with the test result of the batch.                                               |
```

File: content/en/synthetics/cicd_testing/ci_results_explorer.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+| `Test Name`      | The test name associated with the test result of the batch.                                             |
+
+### Create your query
```
Comment (alai97) [inline content/en/synthetics/cicd_testing/ci_results_explorer.md:73]:
```suggestion
### Creating your query
```

File: content/en/synthetics/cicd_testing/ci_results_explorer.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+
+The query syntax of the CI Results Explorer is the same as the one used to query tests on the **Tests** page. Learn more about how to create queries [here][2].
```
Comment (alai97) [inline content/en/synthetics/cicd_testing/ci_results_explorer.md:75]:
```suggestion
To query the CI Results Explorer data, use the [same query syntax][2] as on the **Tests** page.
```

File: content/en/synthetics/troubleshooting/_index.md [hunk 10]
```diff
@@ -138,6 +138,12 @@ This might mean your private location is unable to reach the endpoint your API t
 
 {{< img src="synthetics/timeout.png" alt="API test on private location timing out" style="width:100%;" >}}
 
+## CI/CD Testing
+
+### I'm not seeing any CI metadata in my CI Results Explorer
+
+It might be because you are using API endpoints to trigger your CI/CD test runs. To have your CI Results Explorer populated with CI metadata, you need to use the [NPM package][14].
```
Comment (alai97) [inline content/en/synthetics/troubleshooting/_index.md:145]:
```suggestion
Check whether you are using API endpoints to trigger your CI/CD test runs. To have your CI Results Explorer populate with CI metadata, you must use the [NPM package][14].
```

**Review by alai97 — CHANGES_REQUESTED** [review]
Left some doc feedback.

File: content/en/synthetics/cicd_testing/ci_results_explorer.md [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+
+When clicking on a batch, a side panel containing batch CI metadata as well as batch test results shows up, allowing you to look into all test executions that were performed as part of this batch and pinpoint any test failure. You can then click on any failing test result to be redirected to the detailed test result page and look into the root cause of the issue more thoroughly.
+
+## Search
+
+### Facets and tags
+
+The panel on the left side of the page lists several facets you can use to search through your batches:
+
+* **Batch** facets allow you to filter on attributes of your batches:
```
Comment (alai97) [inline content/en/synthetics/cicd_testing/ci_results_explorer.md:41]:
```suggestion
**Batch** facets allow you to filter on attributes of your batches:
```

File: content/en/synthetics/cicd_testing/ci_results_explorer.md [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
+
+**Batch** facets allow you to filter on attributes of your batches:
+
+| Facet            | Description                                                 |
+|------------------|-------------------------------------------------------------|
+| `Summary Status` | The status of the batch: `Passed`, `Failed`, `In Progress`. |
+| `Duration`       | The overall duration of the batch.                          |
+| `ID`             | The id of the batch.                                        |
+
+* **Git** facets allow you to filter on Git related attributes of your batches:
```
Comment (MargotLepizzera) [inline content/en/synthetics/cicd_testing/ci_results_explorer.md]:
```suggestion
* **Git** facets allow you to filter on Git-related attributes of your batches:
```

File: content/en/synthetics/cicd_testing/ci_results_explorer.md [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+
+### Facets and tags
+
+The panel on the left side of the page lists several facets you can use to search through your batches:
+
+**Batch** facets allow you to filter on attributes of your batches:
+
+| Facet            | Description                                                 |
+|------------------|-------------------------------------------------------------|
+| `Summary Status` | The status of the batch: `Passed`, `Failed`, or `In Progress`. |
```
Comment (MargotLepizzera) [inline content/en/synthetics/cicd_testing/ci_results_explorer.md:47]:
```suggestion
| `Summary Status` | The status of the batch: `Passed`, `Failed`, `In Progress`. |
```

File: content/en/synthetics/cicd_testing/ci_results_explorer.md (hunk 5 above)
Comment (MargotLepizzera) [reply content/en/synthetics/cicd_testing/ci_results_explorer.md:69]: Resolving that one for now as we probably want to keep things consistent with the format we're using on that page https://docs.datadoghq.com/synthetics/search/#facets-and-tags where there's no OR, don't you think?

File: content/en/synthetics/cicd_testing/ci_results_explorer.md (hunk 8 above)
Comment (MargotLepizzera) [reply content/en/synthetics/cicd_testing/ci_results_explorer.md:75]: Same comment as above about keeping the format consistent with https://docs.datadoghq.com/synthetics/search/#create-your-query.

**Review by alai97 — APPROVED** [review]
Looks great!

**Pull request merged**